packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simualting browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
+	 */
+	static isAbleToInteract_(node, eventName) {
+		var currElement = node;
+		var isAble = true;
+		var matchesSelector = 'button, input, select, textarea, fieldset';
+
+		if (eventName === 'click') {
+			while (currElement && dom.match(currElement, matchesSelector) && !dom.match(currElement, 'legend')) {
```

Comment:
You're checking this on the same element you're checking for `button, input, select, textarea, fieldset` right before though, so if the code reaches `!dom.match(currElement, 'legend')` it means that the tag is already one of `button, input, select, textarea, fieldset`, so it can't be `legend` anymore, right?